portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,10 @@
 #greeting-container {
```

Comment:
Do you need to clean #greeting-container up?

Reply:
yes I will do that